astropy/units/astrophys.py

```diff
@@ -43,6 +43,8 @@
 
 def_unit(['lyr', 'lightyear'], (_si.c * si.yr).to(si.m),
          namespace=_ns, prefixes=True, doc="Light year")
+def_unit(['lsec', 'lightsecond'], (_si.c * si.s).to(si.m),
+         namespace=_ns, prefixes=True, doc="Light second")
```

Comment:
I don't think I've ever seen light seconds with prefixes, so I would suggest setting `prefixes=False` (the same is arguably true for `lightyear`, but that is too late now...)

Reply:
Probably as a joke, but I've seen people use kilolightyear and megalightyear in talks 😅

Reply:
Don't encourage them... 😆

Reply:
Admittedly, I had no idea what "prefixes" meant, I just copied from a similar unit 😄

Reply:
@mhvk I don't have a way to squash now, I'm on my phone. Do you have an option to "Squash and Merge"?

Reply:
No, we disabled "Squash and Merge" button because it messed with the backport script. I think this can wait, or we'll be happy to help if you do not want to do it.

Reply:
Oh, ok, whoever does it first then. I'll be able to do it later tonight or tomorrow morning. Thanks!

Reply:
Done

Reply:
Thanks! 😸